src/lib.rs

```diff
@@ -37,35 +37,41 @@
 //! }
 //! # }
 //! ```
+use std::ffi::OsStr;
 use std::io;
 use std::process::{Command, ExitStatus};
-use std::ffi::OsStr;
 
 #[cfg(not(any(target_os = "windows", target_os = "macos")))]
 pub fn that<T: AsRef<OsStr> + Sized>(path: T) -> io::Result<ExitStatus> {
-    let mut last_err: io::Result<ExitStatus> = Err(io::Error::from_raw_os_error(0));
+    let path_ref = path.as_ref();
```

Comment:
I don't see a reason for this alias

Reply:
It's really a small optimization; pre-caching a value that will be used inside a loop. On the other hand of course, `as_ref()` could be one of the cheapest method calls, and the loop only iterates (at most) three times.

I come from a Python background, were function/method calls in general are treated as expensive operations, so if a value can be reused for every iteration of a loop (because the call result is equivalent every time) you would just save the value and use it directly.